src/current/molt/molt-fetch.md

```diff
@@ -503,6 +504,87 @@ The JSON is formatted as follows:
 - `column` specifies the column that will use the custom type mapping in `type-kv`. If `*` is specified, then all columns in the `table` with the matching `source-type` are converted.
 - `type-kv` specifies the `source-type` that maps to the target `crdb-type`.
 
+### Transformations
+
+You can define transformation rules to be performed on the target schema during the fetch process. These can be used to:
+
+- Map [computed columns]({% link {{ site.current_cloud_version }}/computed-columns.md %}) to a target schema.
+- Map [partitioned tables]({% link {{ site.current_cloud_version }}/partitioning.md %}) to a single target table.
+- Rename tables on the target schema.
+
+Transformation rules are specified using a JSON file and `--transformations-file`. For example:
+
+~~~ json
+{
+  "transforms": [
+    {
+      "id": 1,
+      "resource_specifier": {
+        "schema": ".*",
+        "table": ".*"
+      },
+      "column_exclusion_opts": {
+        "add_computed_def": true,
+        "column": "^age$"
+      }
+    },
+    {
+      "id": 2,
+      "resource_specifier": {
+        "schema": "public",
+        "table": "charges_part.*"
+      },
+      "table_rename_opts": {
+        "value": "charges"
+      }
+    }
+  ]
+}
+~~~
+
+- `resource_specifier` configures the following options for transformation rules:
+	- `schema` specifies the schemas to be affected by the transformation rule, formatted as a POSIX regex string.
+	- `table` specifies the tables to be affected by the transformation rule, formatted as a POSIX regex string.
+- `column_exclusion_opts` configures the following options for column exclusions and computed columns:
+	- `column` specifies source columns to exclude from being mapped to regular columns on the target schema. It is formatted as a POSIX regex string.
+	- `add_computed_def`, when set to `true`, specifies that each matching `column` should be mapped to a [computed column]({% link {{ site.current_cloud_version }}/computed-columns.md %}) on the target schema, using the computed column definition on the source schema. This assumes that all matching columns are computed columns on the source.
+		{{site.data.alerts.callout_danger}}
+		Columns that match the `column` regex will **not** be moved to CockroachDB if either `add_computed_def` is omitted, or a matching column is a non-computed column.
+		{{site.data.alerts.end}}
+- `table_rename_opts` configures the following option for table renaming:
+	- `value` specifies the table name to which the matching `table` is mapped. If there is only one matching `table` on the source, it is renamed to `value` on the target. If there is more than one matching `table`, this assumes that all matching tables are [partitioned tables]({% link {{ site.current_cloud_version }}/partitioning.md %}) with the same schema. Otherwise, the process will error.
```

Comment:
```suggestion
	- `value` specifies the table name to which the matching `resource_specifier` is mapped. If there is only one table matching `resource_specifier` on the source, it is renamed to `table_rename_opts.value` on the target. If there is more than one matching `resource_specifier` (i.e. `n` to `1` mapping), this assumes that all matching tables are [partitioned tables]({% link {{ site.current_cloud_version }}/partitioning.md %}) with the same schema. Otherwise, the process will error. 
	- Additionally, in such `n` to `1` mapping situation: 
	     - Since these tables' data will be ingested into the single target table _concurrently_, only `--use-copy` or `--direct-copy` mode is allowed for the data ingestion phase. 
	     - User will have to create the schema of the target table on their own, and must choose `--table-handling=none` mode. I.e. `--table-handling={truncate|drop-on-target-and-recreate}` are disabled for this case.
```